src/plays/popup-component/PopupComponent.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PlayHeader from 'common/playlists/PlayHeader';
+import PopupCard from './components/PopupCard';
+import './styles.css';
+
+// WARNING: Do not change the entry componenet name
+function PopupComponent(props) {
+  return (
+    <>
+      <div className="play-details">
+        <PlayHeader play={props} />
+        <div className="play-details-body">
```

Comment:
The best thing to do is to add an additional class name here and use that as a prefix for all the styles in CSS file.

Reply:
@lifeofsoumya this comment is not taken care. Could you please respond?